ddhcpd/files/etc/ddhcpd.hook

```diff
@@ -2,6 +2,7 @@
 
 for i in /etc/ddhcpd.d/*
 do
-  $i "$@"
+  if [ -e "$i" ] ; then
+    $i "$@"
```

Comment:
If quoting $i in the conditional you should be consistent and quote $i everywhere.